packages/docs-reanimated/docs/guides/contributing.mdx

```diff
@@ -0,0 +1,318 @@
+---
+id: contributing
+title: Contributing
+sidebar_label: Contributing
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Thank you for your interest in contributing to Reanimated! From commenting on and triaging issues, through extending the documentation, to reviewing and sending Pull Requests, all contributions are more than welcome.
+
+## Ways to Contribute
+
+1. **Replying and handling open issues** &ndash; great way to contribute without writing a single line of code is triaging the issues. We often get issues that have generic errors, occur only in very specific cases, do not have proper example or reproducible repository. One of they way to help is preparing and filling those details, which will help other contributors get up to speed with the issue faster.
+
+2. **Helping with documentation** &ndash; helping with the documentation can be as simple as checking for speling and grammer. If you encounter situations that can be explained better or in more details, click **Edit this page** at the bottom of most pages to get started with your own contribution. You can also help updating the page structures from old versions, as some of them are still ported from Reanimated v2 docs. Read more about [helping with documentation](#helping-with-documentation).
+
+3. **Reviewing pull requests** &ndash; reviewing Pull Requests is crucial as it may help catch the corner cases or bugs that the contributor didn't notice. Every review matters as it may help us polish quality of the library.
+
+4. **Contributing to Code** &ndash; contributions to the source code generally come in the form of Pull Requests. By contributing to the code you help us with solving issues, fixing bugs or introducing new, amazing features. If you want to start your adventure with open source it's good idea to take a look at [good first issue](https://github.com/software-mansion/react-native-reanimated/pulls?q=is%3Apr+is%3Aopen+label%3A%22good+first+issue%22) on GitHub. Read more about [contributing to code here](#contributing-to-code).
+
+### Repository overview
+
+The React Native Reanimated repository is a monorepo with the following structure:
+
+```
+├── apps
+│   ├── common-app // shared source code of example apps
+│   ├── paper-example // React Native app wrapper running the Old Architecture for shared examples
+│   ├── fabric-example // React Native app wrapper running the New Architecture for shared examples
+│   ├── macos-example // React Native for MacOS app wrapper for shared example code
+│   ├── next-example // Next.js wrapper for shared example code
+│   ├── tvos-example // React Native for TVOS app wrapper for shared examples
+│   └── web-example // React Native for Web app wrapper for shared examples
+└── packages
+    ├── docs-reanimated // documentation described further in [docs section](#helping-with-documentation)
```

Comment:
when in code block the link is not linking - in a meaning it stays as a text and doesn't transform to link